ddtrace/internal/injection.py

```diff
@@ -30,7 +30,7 @@ def _inject_hook(code, hook, lineno, arg):
     identifier for the hook itself. This should be kept in case the hook needs
     to be removed.
     """
-    for i, instr in enumerate(code):
+    for _i, instr in enumerate(code):
```

Comment:
would
```suggestion
    i = -1
    for i, instr in enumerate(code):
```
be accepted by bugbear?

Reply:
We can leave `i` and I'll mark it as `noqa`